### PR TITLE
test: add test for batched defer with production size of batch

### PR DESCRIPTION
**Pull request by VladasZ**

### CODE REVIEW

File: sweat/src/defer.rs [hunk 1]
```diff
@@ -39,6 +44,9 @@ impl SweatDefer for Contract {
         // These values calculated in `measure_record_batch_for_hold_test` in claim contract.
         let record_batch_for_hold_gas = Gas::ONE_TERA * 8 + Gas(batch_len * ONE_GGAS * 320);
 
+        log_str("holding_account_id");
+        log_str(holding_account_id.as_str());
```
Comment (cak3ninja) [inline sweat/src/defer.rs]: Remove it please

File: sweat/src/integration.rs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    json_types::{U128, U64},
+    near_bindgen,
+};
+use sweat_model::{IntegrationTestMethods, SweatApi};
+
+use crate::{Contract, ContractExt};
+
+#[near_bindgen]
+impl IntegrationTestMethods for Contract {
+    fn calculate_fee_for_batch(&self, batch_size: u32, claim_amount: u32) -> (U128, U128) {
```
Comment (cak3ninja) [inline sweat/src/integration.rs]: -> calculate_payout_with_fee_for_batch

**Review by cak3ninja — CHANGES_REQUESTED** [review]

File: sweat/src/defer.rs (hunk 1 above)
Comment (VladasZ) [reply sweat/src/defer.rs]: Fixed

File: sweat/src/integration.rs (hunk 2 above)
Comment (VladasZ) [reply sweat/src/integration.rs]: Fixed

**Review by cak3ninja — APPROVED** [review]

**Pull request merged**